.github/workflows/test-coverage.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
           covr::to_cobertura(cov)
         shell: Rscript {0}
 
-      - uses: codecov/codecov-action@v4
+      - uses: codecov/codecov-action@v4.5
         with:
           fail_ci_if_error: ${{ github.event_name != 'pull_request' && true || false }}
```

Comment:
Why do we need the part `&& true || false` ? Isn't it the same as just `github.event_name != 'pull_request'` ?

Reply:
ok, after some reading: it seems that boolean input arguments of composite workflows are defined in the GH actions in a weird way (https://github.com/actions/runner/issues/2238). So let's keep it AS IS